src/routes/index.ts

```diff
@@ -38,15 +38,21 @@ const urlToObjectTransformation: Dictionary<(value: any) => any> = {
   latitude: (s) => parseFloat(s),
   longitude: (s) => parseFloat(s),
   zoom: (s) => parseFloat(s),
+  sidebar: (s) => (s === 'true' ? true : false),
```

Comment:
isn't the `true : false` redundant ?
```suggestion
  sidebar: (s) => (s === 'true'),
```
and are you sure this is always received as a string?